aip/general/0203.md

```diff
@@ -171,6 +163,15 @@ A resource with an unordered list **may** return the list in a stable order, or
 
 ## Rationale
 
+### Required set of annotations
```

Comment:
I don't think this section should exist.

Reply:
Can you clarify which section, and clarify why you think it shouldn't exist? If you're referring to the rationale section, there was several requests internally and unanimous support among the current API design team. Public discussion is at: https://github.com/aip-dev/google.aip.dev/pull/1058.